policyfile.go

```diff
@@ -160,6 +162,15 @@ type NodeAttrGrantApp struct {
 	Domains    []string `json:"domains,omitempty" hujson:"Domains,omitempty"`
 }
 
+type Grant struct {
+	Source      []string                    `json:"src,omitempty" hujson:"Src,omitempty"`
+	Destination []string                    `json:"dst,omitempty" hujson:"Dst,omitempty"`
+	IP          []string                    `ip:"dst,omitempty" hujson:"IP,omitempty"`
```

Comment:
I have 0 clue how this works, but this just seems to break the rest of this pattern. Does this need to be fixed?

```suggestion
	IP          []string                    `json:"ip,omitempty" hujson:"IP,omitempty"`
```

Reply:
Fixed. Good catch!